lib/dest/index.js

```diff
@@ -12,11 +12,21 @@ function dest(outFolder, opt) {
   }
 
   function saveFile(file, enc, cb) {
+    /* jshint validthis: true */
+    var stream = this;
+
+    function complete(err, file) {
+      cb(err, file);
+      if (!err) {
```

Comment:
@davidchase After consulting the stream docs that seems like it should work. We should make sure our stream implementation is in line with the latest docs